environment/src/main/java/com/sequenceiq/environment/environment/validation/validators/EncryptionKeyArnValidator.java

```diff
@@ -25,4 +61,32 @@ public ValidationResult validateEncryptionKeyArn(String encryptionKeyArn) {
         }
         return validationResultBuilder.build();
     }
+
+    public ValidationResult validate(EnvironmentValidationDto environmentValidationDto, ParametersDto parametersDto) {
+        EnvironmentDto environmentDto = environmentValidationDto.getEnvironmentDto();
+        ValidationResult.ValidationResultBuilder validationResultBuilder = ValidationResult.builder();
+        String encryptionKeyArn = parametersDto.getAwsParametersDto().getAwsDiskEncryptionParametersDto().getEncryptionKeyArn();
```

Comment:
Validate the key existence only when the `encryptionKeyArn` is present - Need not to make the cloud call if no encryption key is provided by user.

Reply:
I think it would be good to validate that even the AwsParametersDto and the AwsDiskEncryptionParametersDto are not null in this call chain.